src/containers/ContactUs/styles.js

```diff
@@ -7,10 +7,10 @@ export const StyledContainer = styled(Container)({
 export const HeadingWrapper = styled("div")({
   marginBottom: "10px",
 });
-export const Content = styled("p")(({ theme: { colors } }) => ({
+export const Content = styled("p")(({ theme: { colors, fontSizes, fontWeights } }) => ({
   color: colors?.grey,
-  fontSize: "22px",
-  fontWeight: "400",
+  fontSize: fontSizes["2xl"],
+  fontWeight: fontWeights.regular,
 }));
 export const TitleWrapper = styled("div")({
   marginTop: "121px",
```

Comment:
Here again

Reply:
used fontSizes and padding, margin from theme.